stdlib/_heapq.pyi

```diff
@@ -1,18 +1,19 @@
 import sys
+from _typeshed import SupportsRichComparison
 from typing import Any, Final, TypeVar
 
-_T = TypeVar("_T")  # list items must be comparable
+_T = TypeVar("_T", bound=SupportsRichComparison, default=Any)  # list items must be comparable
```

Comment:
Any reason not to use `SupportsRichComparisonT` here?

Reply:
I've run into typecheckers complaining about Unknowns when no default is provided for typevars (treating implicit Any as different from explicit Any) scoped to functions, and function objects aren't subscriptable at runtime. Was out of habit due to this, but I think it would require a case like passing heapq.heappush as a variable for this to come up in practice.

Reply:
🤷 I've gone ahead and swapped to just importing SupportsRichComparisonT here. If I run into an issue with a specific typechecker later, I'll just open it with them.